### PR TITLE
[Orca] Support Ray Runtime in TF2Estimator Examples

### PR DESCRIPTION
**Pull request by sgwhat**
This PR is mainly for supporting ray runtime as backend in TF2Estimator Examples.

- [x] Update examples.
- [x] Test Resnet

### CODE REVIEW

**Comment by sgwhat** [comment]
Waiting for http://10.239.45.10:8081/repository fixed.

File: python/orca/example/learn/tf2/yolov3/yoloV3.py [hunk 1]
```diff
@@ -575,33 +577,36 @@ def main():
...
+                                    "spark.kubernetes.driver.volumes.persistentVolumeClaim."
+                                    "nfsvolumeclaim.options.claimName": "nfsvolumeclaim",
+                                    "spark.kubernetes.driver.volumes.persistentVolumeClaim."
+                                    "nfsvolumeclaim.mount.path": options.nfs_mount_path})
+        elif options.cluster_mode.startswith("yarn"):
+            init_orca_context(cluster_mode=options.cluster_mode, cores=options.cores,
+                            num_nodes=options.worker_num, memory=options.memory,
+                            init_ray_on_spark=True, enable_numa_binding=options.enable_numa_binding,
+                            object_store_memory=options.object_store_memory)
+        elif options.cluster_mode == "spark-submit":
```
Comment (jason-dai) [inline python/orca/example/learn/tf2/yolov3/yoloV3.py:608]: I think the default behavior of `init_orca_context` should be the same as "spark-submit"?

**Comment by sgwhat** [comment]
YoloV3 cannot support ray runtime, because `read_parquet` requires spark context.

File: python/orca/example/learn/tf2/yolov3/yoloV3.py (hunk 1 above)
Comment (sgwhat) [reply python/orca/example/learn/tf2/yolov3/yoloV3.py:608]: > I think the default behavior of `init_orca_context` should be the same as "spark-submit"?

Yes, when `activate spark context`.

Comment (sgwhat) [reply python/orca/example/learn/tf2/yolov3/yoloV3.py:608]: > I think the default behavior of `init_orca_context` should be the same as "spark-submit"?

Only when there is an activate spark context, the default behavior is the same as "spark-submit".

Comment (sgwhat) [reply python/orca/example/learn/tf2/yolov3/yoloV3.py:608]: > I think the default behavior of `init_orca_context` should be the same as "spark-submit"?

Yes, when there is an activate spark context.

**Comment by sgwhat** [comment]
Jenkins passed: http://10.112.231.51:18888/job/BigDL-Orca-PR-Validation/555/

**Pull request merged**